rosdep/python.yaml

```
@@ -5270,6 +5270,16 @@ python-xmlplain-pip:
   ubuntu:
     pip:
       packages: [xmlplain]
+python-xmlschema:
```

If this is a pip only package it should have the `-pip` suffix. 

And while we're at it these days most platforms are on python3 now so we should make this rule more future consistent using python3 in the key as pip will choose the local environment anyway.

```suggestion
python3-xmlschema-pip:
```